<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>informacje</class>
 <widget class="QDialog" name="informacje">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="predkoscL">
     <property name="text">
      <string>Boud Rate:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Parametry">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="ParzystoscW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="odswiez">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="bity_stopuL">
     <property name="text">
      <string>Stop Bits:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="bity_stopuW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QTextBrowser" name="informacja"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="bity_slowa">
     <property name="text">
      <string>Word lenght:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="bity_slowaW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="parzystoscL">
     <property name="text">
      <string>Parity:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="predkoscW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Nazwa_porty_L">
     <property name="text">
      <string>                     Port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="Nazwa_portu_W"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
